docs/getting-started/generating-a-mnemonic.en.md

```diff
@@ -1,4 +1,4 @@
-Krux has support for creating 12 and 24-word mnemonic seed phrases. Since true entropy is difficult to produce, especially with an embedded device, Krux outsources entropy generation to the user.
+Krux has support for creating 12 and 24-word mnemonic seed phrases. Since true entropy is difficult to produce, especially with an embedded device, we recommend to outsources entropy generation using dice rolls, but it is also possible to use camera as source of entropy to quickly create a mnemonic.
```

Comment:
"to outsources entropy" becomes "to outsource entropy"
and
"camera as source of" becomes "camera as a source of"